### DISCUSSION TITLE
How to get the max badges ?

### DISCUSSION THREAD

**Discussion by Tran-Quyen** [community/community#18337] [Other Feature Feedback, Questions, & Ideas] [answered]
Someone can help me for this problem

**Comment by ale-lit** [comment, marked as answer]
Hi! Lots of info on this link: https://github.com/Schweinepriester/github-profile-achievements
Also, some information on obtaining can be found in these discussions: https://github.com/Schweinepriester/github-profile-achievements/discussions

![image](https://user-images.githubusercontent.com/91823697/173179107-08b0d713-b134-401d-adf5-545b7cd6d32d.png)

**Reply by Tran-Quyen** [reply]
Thank you. It so help full

**Comment by Nishant1500** [comment]
You can have a look at this -> https://github.com/Schweinepriester/github-profile-achievements#achievements

![SmartSelect_20220611-132502_Chrome](https://user-images.githubusercontent.com/53296643/173179040-6b4556a6-0c48-4de1-b59a-f7b2506f1fe0.jpg)

**Reply by Tran-Quyen** [reply]
Thank you. I know what to do

**Comment by KunalSingh19** [comment]
Galaxy Brain

After answering two discussions, You will get Galaxy Brain badge

Pull Shark

2 opened pull requests that have been merged

YOLO

Merged pull request without code review

**Reply by Tran-Quyen** [reply]
Thank you. It very easy to know how to get them

**Comment by Jino0910** [comment]
사람들이 많이 친절하네
보기 좋아 :+1:

**Reply by Tran-Quyen** [reply]
Thank for your feed back
